### DISCUSSION TITLE
Using the API with our user account.

### DISCUSSION THREAD

**Discussion by Eikosa** [jheddings/notional#112] [Q&A] [answered]
How can we use the api with our user account? I tried to do it with token v2 but it didn't work

**Comment by jheddings** [comment, marked as answer]
Hi @Eikosa !  Thanks for giving Notional a spin...

You'll need to set up an integration in your account (see [here](https://developers.notion.com/docs/authorization) for details).  The official API does not work with the legacy `token_v2` method.

Once you have an integration token, you can pass that to the Notional session and off you go.  Check out the [examples](https://github.com/jheddings/notional/tree/main/examples) for some ideas to get you started.

**Comment by Eikosa** [comment]
> Hi @Eikosa ! Thanks for giving Notional a spin...
> 
> You'll need to set up an integration in your account (see [here](https://developers.notion.com/docs/authorization) for details). The official API does not work with the legacy `token_v2` method.
> 
> Once you have an integration token, you can pass that to the Notional session and off you go. Check out the [examples](https://github.com/jheddings/notional/tree/main/examples) for some ideas to get you started.

![image](https://user-images.githubusercontent.com/20538090/211679298-86580c2e-fd37-4e15-89f1-6aefcbb6829d.png)
I don't know why but it can't access my integration pages and gives this error
![image](https://user-images.githubusercontent.com/20538090/211679405-5087d196-8dec-4164-a044-2fa47f30fd33.png)

**Comment by jheddings** [comment]
That error usually means your page is not shared with the integration...  You have to grant explicit permission for each integration to interact with your content.  Use the "Share" menu on your intended page and add your integration.  When you share a page, all its children will be available to the integration.

**Comment by Eikosa** [comment]
oh yes I got it, thank you so much! i will show where to look for others who also needs for this
![image](https://user-images.githubusercontent.com/20538090/211681640-735f8c39-46ba-44da-9089-ddda849148c0.png)
